src/plugins/odyssey/declaration.js

```diff
@@ -0,0 +1,83 @@
+// @flow
+
+import {EdgeAddress, NodeAddress} from "../../core/graph";
+
+import type {PluginDeclaration} from "../../analysis/pluginDeclaration";
+import type {NodeType} from "../../analysis/types";
+
+export type OdysseyNodeTypeIdentifier =
```

Comment:
Maybe add a TODO to the code that documents that we want to switch the node types into the instance itself (eventually, nonurgent while SC is primary consumer.)